components/example-notebook-servers/codeserver/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/base:master-6c41e72b
+ARG BASE_IMG=public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/base:master-6c41e72b
```

Comment:
Same for here as well:
1. `BASE_IMG=<base>`
2. A comment to instruct users to use the makefile to build the image

Reply:
Fixed!